addons/account/views/account_move_views.xml

```diff
@@ -844,6 +848,10 @@
                                             <label for="name" string="Section" attrs="{'invisible': [('display_type', '!=', 'line_section')]}"/>
                                             <label for="name" string="Note" attrs="{'invisible': [('display_type', '!=', 'line_note')]}"/>
                                             <field name="name" widget="text"/>
+                                            <group>
+                                                <field name="price_subtotal" string="Subtotal"/>
+                                                <field name="price_total" string="Total"/>
```

Comment:
Aren't you missing something else now? :wink: 

https://github.com/odoo/odoo/blob/5b95c97a85ee93f690097b183a39d8e84fa92f65/addons/account/views/account_move_views.xml#L715-L720

https://github.com/odoo/odoo/blob/5b95c97a85ee93f690097b183a39d8e84fa92f65/addons/account/views/account_move_views.xml#L749-L750